controller/config.go

```diff
@@ -4,15 +4,17 @@ import (
 	"fmt"
 	"io/ioutil"
 
+	"github.com/golang/glog"
 	"github.com/pkg/errors"
 	"gopkg.in/v2/yaml"
 )
 
 func LoadFile(path string) (*Config, error) {
 	content, err := ioutil.ReadFile(path)
 	if err != nil {
-		return nil, errors.Wrap(err, "failed to read config file")
+		glog.Infof("No %s file found.  Will try to figure out defaults", path)
 	}
+
 	c, err := Load(string(content))
```

Comment:
sorry not completely following this one, can you elaborate?  we don't use auto anymore as we try to work out the automated defaults if domain or exposer are empty.  I've tested this and it seems to be working fine with 
- empty domain + exposer 
- no config.xml 
- populated config.xml using domain and exposer overrides

all seems to be working great